src/main/java/com/google/api/codegen/php/PhpGapicContext.java

```diff
@@ -175,86 +175,85 @@ public String getDescription(ProtoElement element) {
   // =========
 
   /**
-   * A set of PHP keywords and built-ins.
-   * keywords: http://php.net/manual/en/reserved.keywords.php
+   * A set of PHP keywords and built-ins. keywords: http://php.net/manual/en/reserved.keywords.php
    */
   private static final ImmutableSet<String> KEYWORD_BUILT_IN_SET =
       ImmutableSet.<String>builder()
-      .add("__halt_compiler",
-          "abstract",
-          "and",
-          "array",
-          "as",
-          "break",
-          "callable",
-          "case",
-          "catch",
-          "class",
-          "clone",
-          "const",
-          "continue",
-          "declare",
-          "default",
-          "die",
-          "do",
-          "echo",
-          "else",
-          "elseif",
-          "empty",
-          "enddeclare",
-          "endfor",
-          "endforeach",
-          "endif",
-          "endswitch",
-          "endwhile",
-          "eval",
-          "exit",
-          "extends",
-          "final",
-          "finally",
-          "for",
-          "foreach",
-          "function",
-          "global",
-          "goto",
-          "if",
-          "implements",
-          "include",
-          "include_once",
-          "instanceof",
-          "insteadof",
-          "interface",
-          "isset",
-          "list",
-          "namespace",
-          "new",
-          "or",
-          "print",
-          "private",
-          "protected",
-          "public",
-          "require",
-          "require_once",
-          "return",
-          "static",
-          "switch",
-          "throw",
-          "trait",
-          "try",
-          "unset",
-          "use",
-          "var",
-          "while",
-          "xor",
-          "yield",
-          "__CLASS__",
-          "__DIR__",
-          "__FILE__",
-          "__FUNCTION__",
-          "__LINE__",
-          "__METHOD__",
-          "__NAMESPACE__",
-          "__TRAIT__")
-      .build();
-
+          .add(
```

Comment:
Looks like @bjwatson did the mass-reformatting (Apr. 20) before this was added (May 5)